backend/routes.js

```diff
@@ -17,16 +17,15 @@ import {
 import QueryScheduler from './persistent/QueryScheduler.js';
 import {getSetting, saveSetting} from './settings.js';
 import {checkWritePermissions} from './persistent/PlotlyAPI.js';
-import {contains, has, isEmpty, merge, pluck} from 'ramda';
+import {contains, has, keys, isEmpty, merge, pluck} from 'ramda';
 import {getCerts, fetchAndSaveCerts, setRenewalJob} from './certificates';
 import Logger from './logger';
 import fetch from 'node-fetch';
 
-const opn = require('opn');
 const HOSTS = '/etc/hosts';
 
 export default class Server {
-    constructor(args = {}) {
+    constructor(args = {createCerts: true}) {
```

Comment:
see earlier comments about moving some of this logic out of the constructor so that initializing the `Server` has less side affects